EasyCommands/Common/Operations.cs

```diff
@@ -89,11 +89,11 @@ partial class Program {
             AddBiOperation<bool, bool>(BiOperand.COMPARE, (a, b) => a.CompareTo(b));
             AddBiOperation<string, string>(BiOperand.COMPARE, (a, b) => a.CompareTo(b));
             AddBiOperation<float, float>(BiOperand.COMPARE, (a, b) => a.CompareTo(b));
-            AddBiOperation<Vector3D, Vector3D>(BiOperand.COMPARE, (a, b) => a.Equals(b));
+            AddBiOperation<Vector3D, Vector3D>(BiOperand.COMPARE, (a, b) => !a.Equals(b));
             AddBiOperation<Color, Color>(BiOperand.COMPARE, (a, b) => a.PackedValue.CompareTo(b.PackedValue));
             AddBiOperation<Vector3D, float>(BiOperand.COMPARE, (a, b) => a.Length().CompareTo(b));
-            AddBiOperation<float, Vector3D>(BiOperand.COMPARE, (a, b) => a.CompareTo(b.Length()));
-            AddBiOperation<KeyedList, KeyedList>(BiOperand.COMPARE, (a, b) => Enumerable.SequenceEqual(a.keyedValues, b.keyedValues));
+            AddBiOperation<float, Vector3D>(BiOperand.COMPARE, (a, b) => a.CompareTo((float)b.Length()));
```

Comment:
is the float cast actually necessary? if so we likely need it on line 94 as well

Reply:
`b.Length()` returns `double` and `CompareTo(...)` has no overloads for narrowing conversions.
On line 94 is is not strictly necessary and in most cases makes no difference but if you want these to be interchangable it should be cast to `float` as well.

Reply:
ya we should make both of them cast to float in that case.

Reply:
Both cases use `CastNumber(ResolvePrimitive(...))` now, that should be 1 resp. 3 symbols shorter than the explicit cast after minifying.